src/prng/hc128.rs

```diff
@@ -19,21 +19,21 @@ use impls;
 /// A cryptographically secure random number generator that uses the HC-128
 /// algorithm.
 ///
-/// HC-128 is a stream cipher designed by Hongjun Wu[1], that we use as an RNG.
+/// HC-128 is a stream cipher designed by Hongjun Wu [1], that we use as an RNG.
 /// It is selected as one of the "stream ciphers suitable for widespread
-/// adoption" by eSTREAM[2].
+/// adoption" by eSTREAM [2].
 ///
 /// HC-128 is an array based RNG. In this it is similar to RC-4 and ISAAC before
 /// it, but those have never been proven cryptographically secure (or have even
-/// been broken).
+/// been sgnificantly coompromised, as in the case of RC-4 [5]).
```

Comment:
typo

Reply:
I have my first mechanical keyboard now, and it takes some getting used to :smile: